### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by aster-alemu**
In this PR I have been doing the following activities

1. Create remote repository
2. Clone remote repository on to my local machine using git clone command followed by the SSH link
3. Set up GitHub action and Linters on my local environment
4. Create branch repository using the command git branch followed by the name of the new branch
5. Switch from main branch to newly created branch using the command git checkout followed by the name of newly created branch
6. Create html file with name index.html
7. Create css file with name style.css
8. Edit the README.md file
9. Check for stylelint error
10. Add, commit and push the file into Mobile version branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        <li><a href="https://github.com/aster-alemu"><i class="fa-brands fa-github"></i></a></li> 
+        <li><a href="https://linkedin.com/in/aster-alemu"><i class="fab fa-linkedin-in"></i></a></li>  
+        <li><a href=""><i class="fa-solid fa-hand-peace"></i></a></li> 
+        <li><a href="https://twitter.com/asteralemu"><i class="fab fa-twitter"></i></a></li>
+        <li><a href=""><i class="fa-solid fa-m"></i></a></li> 
+        <li><img src="image/background-image.png" alt="hero image"></li>
+      </ul>
+    </section>
+  </div>
+  <div class="works">
```
Comment (zieeco) [inline index.html]: - [ ] Please, kindly use a `<section tag>` instead of `<div>` tag here.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+  font-size: 15px;
+  margin-top: 12px;
+  padding: 5px;
+}
+
+.cards .button5 {
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 10px;
+}
```
Comment (zieeco) [inline style.css]: - [ ] Please, kindly use the right `font-family` on your `Multi-Post stories`
- [ ] Add `margin-top, margin-bottom`  to your `paragraph` and `header` 
- [ ] Align, your `see-project-button` and `list items buttons` to match the design. This happened as a result of the padding you add to the `li` in your `style.css` line `80` 

```
li {
  display: inline;
  list-style: none;
  padding: 12px;
}
```
> || **YOUR DESIGN** || **FIGMA DESIGN** ||
> |-|---------|-|---------|-|
> || section  ||  section2 ||

File: style.css [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+.cards h1 {
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  color: #675a53;
+  padding-top: 120px;
+  padding-bottom: 24px;
+}
```
Comment (zieeco) [inline style.css]: - [ ] Kindly, use the color, font family,  and font-weight from the Figma design for your `My Recent Work`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+.cardtwo h5 {
+  font-family: Crete Round, sans-serif;
+  font-size: 32px;
+  color: #fff;
+  font-style: normal;
+  padding-left: 24px;
+  padding-top: 120px;
+  padding-bottom: 24px;
+}
```
Comment (zieeco) [inline style.css:196]: - [ ]  Kindly, use the font family from the Figma design

File: style.css [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+  display: block;
+}
+
+.heading {
+  font-family: InaiMathi, sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  text-align: center;
+}
```
Comment (zieeco) [inline style.css]: - [ ] Please, kindly add the `icons` to your `headings`



one

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @aster-alemu 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice design
- Descriptive PR
- No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @aster-alemu, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] The button (e.g., "See project, Get my resume, submit") is a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design), Kindly add this before further project building.
- 
Screenshot from 2022-04-23 14-17-17

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Deepakdanger** [comment]
I mistakenly, select approve here, but you can ask for review after completing the required changes.

**Don't merge now, Ask for a review after doing changes**

**Review by roshan-bajgain — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 

## Optional Suggestion

- [ ] This is your last review So I am providing you optional feedback Kindly implement it: When you `pressed`  your button and when you `Enabled` your button it is working fine but when the user ` Hover` the button then your design is not like Figma. so see this image below
### Your button when hover

your hover

### Figma design
Figma


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**